app/assets/templates/pin-show.html

```diff
@@ -17,7 +17,7 @@
           
         <div class="image-content">
           <a href="{{ pin.source_url }}">
-            <img ng-src='{{ pin.img_sm_url }}' class="big-image">
+            <img src='{{ pin.img_sm_url }}' class="big-image">
```

Comment:
Ng breaks the conditional function for no scrolling function when modal
pops up
 On 5 Feb 2015 09:23, "Paul" <notifications@github.com> wrote:

> In app/assets/templates/pin-show.html
> <https://github.com/gduck/fritter/pull/79#discussion_r24136736>:
>
> > @@ -17,7 +17,7 @@
> >
> >          <div class="image-content">
> >            <a href="{{ pin.source_url }}">
> > -            <img ng-src='{{ pin.img_sm_url }}' class="big-image">
> > +            <img src='{{ pin.img_sm_url }}' class="big-image">
>
> Why remove ng- ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gduck/fritter/pull/79/files#r24136736>.
>